spec/features/user_login_spec.rb

```diff
@@ -17,7 +17,7 @@
     it 'allows the user to logout in' do
       click_link_or_button 'Sign out'
       expect(page).to have_content('You have successfully signed out.')
-      expect(page).to have_content('Sign in')
+      expect(page).to have_content('Email')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.